dashboards-observability/public/components/explorer/home_table/history_table.tsx

```diff
@@ -0,0 +1,115 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+import React, { useState, useRef, useEffect } from 'react';
+
+import {
+  EuiBasicTable,
+  EuiSpacer,
+  EuiLink,
+} from '@elastic/eui';
+import { uniqueId, get } from 'lodash';
+import { useDispatch } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import { changeQuery } from '../slices/query_slice';
+import { initialTabId } from '../../../framework/redux/store/shared_state';
+import { RAW_QUERY } from '../../../../common/constants/explorer';
+
+interface TableData {
+  savedHistory: any;
+  savedQuerySearch: (searchQuery: string, selectedDateRange: [], selectedTimeStamp, selectedFields: []) => void;
+  // savedQueryChange: (query: string, index: string) => void;
+  // savedTimeChange: (timeRange: Array<string>) => void;
+}
+
+export function Table(options: TableData) {
+  const [pageIndex, setPageIndex] = useState(0);
+  const [pageSize, setPageSize] = useState(10);
+  const dispatch = useDispatch();
+  const history = useHistory();
+  const hisRef = useRef();
+  hisRef.current = pageIndex;
+  const thisRef = useRef();
+  thisRef.current = pageSize;
+
+  // const query= "search source=opensearch_dashboards_sample_data_logs | where utc_time > timestamp('2021-07-01 00:00:00') and utc_time < timestamp('2021-07-02 00:00:00')";
```

Comment:
Minor: we can remove comments here as well.

Reply:
I'll remove them